test/parallel/test-cluster-bind-privileged-port.js

```diff
@@ -24,12 +24,16 @@ const common = require('../common');
 const assert = require('assert');
 const cluster = require('cluster');
 const net = require('net');
-const { readFileSync } = require('fs');
+const { readFileSync, statSync } = require('fs');
 
 if (common.isLinux) {
-  const unprivilegedPortStart = parseInt(readFileSync('/proc/sys/net/ipv4/ip_unprivileged_port_start'));
-  if (unprivilegedPortStart <= 42) {
-    common.skip('Port 42 is unprivileged');
+  const procFileName = '/proc/sys/net/ipv4/ip_unprivileged_port_start';
+  // Does not exist for Kernel < 4.1 where answer is 1024. So only test limit if limit exists
+  if (statSync(procFileName, { throwIfNoEntry: false })) {
+    const unprivilegedPortStart = parseInt(readFileSync(procFileName));
+    if (unprivilegedPortStart <= 42) {
+      common.skip('Port 42 is unprivileged');
+    }
```

Comment:
This is a TOCTOU pattern. While mostly academic in this context, node users sometimes copy "best practices" from the test suite so I'd rather not merge this code as-is. Open the file and handle the error when it's not there.

Reply:
Agreed. While the TOCTOU race condition can never occur here, your reasoning is all sound. I'll replace it with a try block. Just dislike empty catch blocks but we'll throw a comment in there.
Done.